utils/scandetails.go

```diff
@@ -177,7 +188,10 @@ func (sc *ScanDetails) RunInstallAndAudit(workDirs ...string) (auditResults *res
 		SetInstallCommandArgs(sc.InstallCommandArgs).SetUseJas(true).
```

Comment:
remove SetUseJas(true) here:
```suggestion
		SetInstallCommandArgs(sc.InstallCommandArgs).
```